### PR TITLE
Add shortcut hints.

### PR DESCRIPTION
**Pull request by adamjspooner**
@keybase/react-hackers, this is ready for review.

### CODE REVIEW

File: shared/chat/conversation-filter-input.js [hunk 1]
```diff
@@ -2,6 +2,9 @@
 import * as React from 'react'
 import * as Kb from '../common-adapters'
 import * as Styles from '../styles'
+import * as Platforms from '../constants/platform'
+
+const platformShortcutSymbol = Platforms.isDarwin ? '⌘' : '^'
```
Comment (adamjspooner) [inline shared/chat/conversation-filter-input.js]: Where should I put this? We'll be using it more often moving forward to teach people about keyboard shortcuts.

File: shared/chat/conversation-filter-input.js [hunk 2]
```diff
@@ -113,6 +116,9 @@ class ConversationFilterInput extends React.PureComponent<Props, State> {
           <Kb.Text type="BodySemibold" style={styles.text}>
             Jump to chat
           </Kb.Text>
+          <Kb.Text type="BodySemibold" style={styles.textFaint}>
```
Comment (buoyad) [inline shared/chat/conversation-filter-input.js]: `{!isMobile &&`

File: shared/chat/conversation-filter-input.js (hunk 1 above)
Comment (buoyad) [reply shared/chat/conversation-filter-input.js]: `constants/platform` seems like a good place

File: shared/chat/conversation-filter-input.js (hunk 2 above)
Comment (adamjspooner) [reply shared/chat/conversation-filter-input.js]: Good catch!

**Pull request merged**